xonsh/proc.py

```diff
@@ -2356,8 +2356,6 @@ def returncode(self):
             return 1
         return self.proc.returncode
 
-    rtn = returncode
```

Comment:
Please don't remove this API

Reply:
there is already a property with the same name. that was error from mypy

Reply:
?

Reply:
Oh you are right, thanks!